### DISCUSSION TITLE
Published a new article introducing Avalonia 11 and comparing it against Uno and MAUI platforms`

### DISCUSSION THREAD

**Discussion by npolyak** [AvaloniaUI/Avalonia#12611] [General]
[Multiplatform XAML/C# Miracle Package: Avalonia. Comparing Avalonia to WinUI based Solutions](https://www.codeproject.com/Articles/5366945/Multiplatform-XAML-Csharp-Miracle-Package-Avalonia). 

Please, remember that every non-negative comment and upvote will increase the readership of this article and will make Avalonia better known and better received.

**Comment by rabbitism** [comment]
TBH I really doubt the stats of LOLs comparison, they are not benchmarked in the same environment. I never advertise Avalonia with this data.

**Reply by npolyak** [reply]
Thanks, I removed the LOLs comparison chart since there are some doubts about it.

**Comment by BAndysc** [comment]
> [Avalonia XPF](https://avaloniaui.net/XPF) - a commercial version of Avalonia, pretty expensive, but apparently compatible with Telerik, DevEx, ActiPro, Infragistics, Syncfusion and SciChart controls.

With all my heart to Avalonia. The sentence from the article is not true, Avalonia XPF is not a commercial version of Avalonia, it is a completely different product - you can say it is a multiplatform wpf fork, a drop in replacement for wpf.

**Reply by npolyak** [reply]
Thanks, I removed the reference to XPF from the article. My impression was that it was built on top of Avalonia and had some extra features making it compatible with WPF. Probably I was wrong.
